### PR TITLE
Add support for Alma Linux

### PR DESCRIPTION
**Pull request by Justin-Fernbaugh**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -8,5 +8,6 @@
 description      'Installs/Configures yum-osuosl'
 version          '2.0.0'
 
-supports 'centos', '~> 7.0'
-supports 'centos', '~> 8.0'
+supports         'almalinux', '~> 8.0'
+supports         'centos', '~> 7.0'
+supports         'centos', '~> 8.0'
```
Comment (ramereth) [inline metadata.rb:13]:
```suggestion
supports         'centos_stream', '~> 8.0'
```

**Review by ramereth — CHANGES_REQUESTED** [review]

File: metadata.rb (hunk 1 above)
Comment (Justin-Fernbaugh) [reply metadata.rb]: > 

Added the changes for centos_stream support

**Review by ramereth — APPROVED** [review]
Patch bump

**Comment by Justin-Fernbaugh** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v2.0.1.  Have a nice day!

**Pull request merged**